fv/wireguard_test.go

```diff
@@ -109,6 +128,12 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ WireGuard-Supported", []api
 	})
 
 	AfterEach(func() {
+		if dmesgKill != nil {
```

Comment:
Do you want to do this only if the test has failed?

Reply:
We definitely want to clean up the process regardless.  We could do the `log` lines only in case of failure, but I think we're covered by the fact that logging is only displayed (by default) when a test fails, and it can be useful to see the logging even when a test is passing, by running with `-ginkgo.v`.